docs/deploy-project-contracts.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Deploy a Flow Project
+sidebar_title: Deploy a Project
+description: How to deploy Flow project contracts with the CLI
+---
+
+⚠️ _Warning: CLI projects are an experimental feature._
+
+```shell
+flow project deploy
+```
+
+This command automatically deploys your project's contracts based on the 
+configuration defined in your `flow.json` file.
+
+Before using this command, read about how to 
+[configure project contracts and deployment targets](https://docs.onflow.org/flow-cli/project-contracts/).
+
+## Example Usage
+
+```shell
+# Deploy project contracts to all Testnet targets
+> flow project deploy --network=testnet
+
+Deploying Foo to my-testnet-account (0xf8d6e0586b0a20c7)...
+Deploying Bar to my-testnet-account (0xf8d6e0586b0a20c7)...
+
+All contracts deployed successfully.
+```
+
+In the example above, your `flow.json` file might look something like this:
+
+```json
```

Comment:
add `...` before and after json config example so we show that is not complete flow.json.